src/panels/config/person/ha-config-person.ts

```diff
@@ -48,9 +51,12 @@ class HaConfigPerson extends LitElement {
     }
     const hass = this.hass;
     return html`
-      <hass-subpage
-        .header=${hass.localize("ui.panel.config.person.caption")}
-        .showBackButton=${!this.isWide}
+      <hass-tabs-subpage
+        .hass=${this.hass}
+        .narrow=${this.narrow}
+        .route=${this.route}
+        back-path="/config"
+        .tabs=${configSections[2]}
```

Comment:
Any reason it's a list and not independent exports NAV_INTEGRATIONS etc. List indices are fragile as you might end up changing the entries and now you need to update a lot of places, and typescript can't tell us if we messed up .